### PR TITLE
Updates utils.js by adding `PATCH` support

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Contributes to https://github.com/elastic/docs/issues/2966

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
@elasticmachine run docs-build rebuild

File: resources/web/docs_js/utils.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export function console_regex() {
   // https://github.com/elastic/elasticsearch/blob/master/buildSrc/src/main/groovy/org/elasticsearch/gradle/doc/RestTestsFromSnippetsTask.groovy#L71-L79
   var method = '(GET|PUT|POST|HEAD|OPTIONS|DELETE)';
```
Comment (nastasha-solomon) [inline resources/web/docs_js/utils.js:28]:
```suggestion
  var method = '(GET|PUT|POST|PATCH|HEAD|OPTIONS|DELETE)';
```

**Comment by nastasha-solomon** [comment]
I checked the following pages and the `Copy as curl` function works correctly:
- https://docs_bk_2967.docs-preview.app.elstc.co/guide/en/kibana/current/cases-api-update-comment.html#_examples_33
- https://docs_bk_2967.docs-preview.app.elstc.co/guide/en/kibana/current/cases-api-update-comment.html#_examples_33
- https://docs_bk_2967.docs-preview.app.elstc.co/guide/en/security/current/rules-api-update.html#_example_request_3

**Review by lcawl — APPROVED** [review]
Double-checked on some older versions and it works there too (e.g. https://docs_bk_2967.docs-preview.app.elstc.co/guide/en/security/7.17/cases-api-update.html)

**Pull request merged**